app/komponenter/samtykkepanel/SamtykkePanel.tsx

```diff
@@ -0,0 +1,57 @@
+import { ConfirmationPanel } from '@navikt/ds-react';
+import { useState } from 'react';
+import { SanityDokument } from '~/typer/sanity/sanity';
+import TekstBlokk from '../tekstblokk/TekstBlokk';
+import css from './samtykkepanel.module.css';
+import { TypografiTyper } from '~/typer/typografi';
+
+interface Props {
+  tittel: SanityDokument;
+  innhold: SanityDokument;
+  samtykke: SanityDokument;
+  feilmelding: SanityDokument;
+  vedSamtykkeEndring: (bekreftet: boolean) => void;
+  feilmeldingAktivert: boolean;
+}
+
+const SamtykkePanel: React.FC<Props> = ({
+  tittel,
+  innhold,
+  samtykke,
+  feilmelding,
+  vedSamtykkeEndring,
+  feilmeldingAktivert,
+}: Props) => {
+  const [samtykkeErBekreftet, settSamtykkeErBekreftet] = useState(false);
```

Comment:
Jeg synes boolske verdier burde starte med `er` de stedene det er mulig. 
`erSamtykkeBekreftet` og `settErSamtykkeBekreftet`

https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
Eg foreslo dette: https://github.com/bekk/nav-familie-endringsmelding/pull/33#discussion_r1243405226